src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -332,3 +332,13 @@ def constant(
                     _value, _shared_memory = _value.astype(_dtype), False
     # Create Constant itself:
     return Constant(_value, shared_memory=_shared_memory)
+
+
+@unary_op
+def result(data: Node, name: Optional[str] = None) -> Node:
+    """Return a node which represents an output of a graph (Model).
+
+    :param data: The tensor containing the input data
+    :return: Result node
+    """
+    return Result(data.output(0))
```

Comment:
Since the opset1 `result` op was created by node factory instead usage of explicit py bindings of `Result` from `openvino.runtime.op`.

Is there any difference or gain by such change?
https://github.com/openvinotoolkit/openvino/blob/fbc3b251c67a961db6a0325d389c2b53fff1b9a7/src/bindings/python/src/openvino/runtime/opset1/ops.py#L2638-L2645